server/hedley/modules/custom/hedley_general/hedley_general.module

```diff
@@ -5,6 +5,10 @@
  * Code for the Hedley General feature.
  */
 
+// Maximal number of days that may pass between the date on which test
+// execution was recorded, and the date when the results should be set.
+define('HEDLEY_GENERAL_LAB_EXPIRATION_PERIOD', 35);
```

Comment:
It is not one month, but not that far. Is it a business requirement?

Reply:
Yes, per customer requirements.